composer.json

```diff
@@ -26,6 +26,7 @@
         "php": ">=7.1.3",
         "ext-json": "*",
         "doctrine/collections": "^1.5",
+        "laminas/laminas-zendframework-bridge": "^1.1",
         "myclabs/php-enum": "^1.5",
         "ocramius/proxy-manager": "^2.1",
```

Comment:
lets move to `friendsofphp/proxy-manager-lts`. It's a replacement for `ocramius/proxy-manager` which supports a wider range of PHP versions and it's currently being used by symfony.